### PR TITLE
docs(dpg): update old terminology

### LINKED ISSUE #26453: Rename LLC to Data Plane Generated Client
See details in https://github.com/azure/autorest.csharp/issues/1699.  We will do the same for the azure-sdk-for-net repo

### LINKED ISSUE #26543: The permissions parameter description is wrong.
[Enter feedback here]
The permissions parameter description incorrectly duplicates the expiresOn parameter description.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 30d49f9c-4722-9d8e-0cbc-9f7dd7f7a7d2
* Version Independent ID: 4a433799-1586-9819-9496-9de9d23001db
* Content: [QueueSasBuilder Constructor (Azure.Storage.Sas) - Azure for .NET Developers](https://docs.microsoft.com/en-us/dotnet/api/azure.storage.sas.queuesasbuilder.-ctor?view=azure-dotnet#azure-storage-sas-queuesasbuilder-ctor(azure-storage-sas-queueaccountsaspermissions-system-datetimeoffset))
* Content Source: [xml/Azure.Storage.Sas/QueueSasBuilder.xml](https://github.com/Azure/azure-docs-sdk-dotnet/blob/master/xml/Azure.Storage.Sas/QueueSasBuilder.xml)
* Service: **azure**
* GitHub Login: @rloutlaw
* Microsoft Alias: **routlaw**

### PR DESCRIPTION
**Pull request by archerzz**
renaming some deprecated names as follows:
- LLC clients - (Data plane) generated clients
- LLC + grow up APIs - (Data plane) convenience clients
- HLC - (Gen 1) convenience clients
- MPC - (Management plane) generated clients

resolve #26453

# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

**Review by lirenhe — COMMENTED** [review]
@archerzz, it seems this PR is not related to https://github.com/Azure/azure-sdk-for-net/issues/26543, did you link the correct issue in the description?

**Comment by archerzz** [comment]
> @archerzz, it seems this PR is not related to #26543, did you link the correct issue in the description?

@lirenhe Sorry, it's a typo. I've connected to the correct issue.

**Review by lirenhe — APPROVED** [review]
LGTM

File: sdk/core/Azure.Core/tests/LowLevelClientTests.cs [hunk 1]
```diff
@@ -102,7 +102,7 @@ public void CannotGetOutputModelOnFailureCodeAsync()
         //    ResponseClassifier, which will become the only path after resolution of #24031
```
Comment (annelo-msft) [inline sdk/core/Azure.Core/tests/LowLevelClientTests.cs:102]: It would be good to change the name of these files as well.

Comment (archerzz) [reply sdk/core/Azure.Core/tests/LowLevelClientTests.cs:102]: @annelo-msft Done. Rename `LowLevelClientTests` to `DataPlaneGeneratedClientTests`.

**Review by annelo-msft — APPROVED** [review]

**Pull request merged**